tests/libcxx/usual_matrix.lst

```diff
@@ -3,7 +3,7 @@
 
 RUNALL_INCLUDE ..\universal_prefix.lst
 RUNALL_CROSSLIST
-PM_CL="/EHsc /MTd /std:c++latest /permissive- /FImsvc_stdlib_force_include.h /wd4643"
+PM_CL="/EHsc /MTd /std:c++latest /permissive- /FImsvc_stdlib_force_include.h /D_SILENCE_NONFLOATING_COMPLEX_DEPRECATION_WARNING /wd4643"

```

Comment:
We can push this change into LLVM after we otherwise finish review of the PR. I'd hate for someone to make the change and then have to do so again if we decide to change the name of the suppression macro.

Reply:
Upon further consideration, I don't want to block this waiting for the `ranges::is_permutation` PR (which includes the pertinent LLVM changes) to merge. I'd be happy to merge this as-is with an issue to clean it up once we're able to update LLVM again.